src/riak_kv_qry_compiler.erl

```diff
@@ -518,7 +543,7 @@ find_quantum_field_index_in_key2([_|Tail], Index) ->
     find_quantum_field_index_in_key2(Tail, Index+1).
 
 %%
-hash_timestamp_to_quanta(QField, QSize, QUnit, QIndex, MaxSubQueries, Where1) ->
+hash_timestamp_to_quanta(QField, QSize, QUnit, QIndex, Where1) ->
```

Comment:
The max subqueries check was reinstated in bfb275b5 (initially in 3fc2471a, it was removed); it is now done earlier in the call chain. In 0c15cd06, it is made configurable in riak_kv.schema.